### PR TITLE
change from DNS to healthcheck

### PR DESCRIPTION
**Pull request by mjudeikis**
move to healthz checking for api server availability. 

@kargakis this still fails with random other errors sometimes. I think this is the best way for now for dev and test. What do you think?

### CODE REVIEW

File: hack/create.sh [hunk 1]
```diff
@@ -81,10 +81,9 @@ az group deployment create -g $RESOURCEGROUP -n azuredeploy --template-file _dat
 
 if [[ "$RUN_SYNC_LOCAL" == "true" ]]; then
     # will eventually run as an HCP pod, for development run it locally
+    # sleep until FQDN/healthz responds with OK. It takes up to 5 min for API server to respond. 
     FQDN=$(awk '/^  fqdn:/ { print $2 }' <_data/manifest.yaml)
-    # sleep until FQDN name is registered in Azure DNS
-    # 6 coresponds to "Could not resolve host"
-    while [[ "$(curl -s -k https://${FQDN} ; echo $?)" == "6" ]]; do sleep 5; done
+    while [[ "$(curl -s -k https://${FQDN}/healthz)" != "ok" ]]; do sleep 5; done
```
Comment (kargakis) [inline hack/create.sh]: use /healthz/ready over /healthz?

Comment (pweil-) [reply hack/create.sh]: +1 to healthz/ready.  I believe healthz could report more useful information if not ready (in the body) but I think it is possible to be healthy and not ready

**Pull request merged**